app/views/collections/show.html.erb

```diff
@@ -100,7 +100,14 @@
 
             <div id="<%= modal_id %>" class="eui-modal-content confirm-modal">
               <a href="#" class="modal-close float-r"><i class="fa fa-times"></i><span class="is-invisible">Close</span></a>
-              <%= form_tag(ingest_specific_granule_path(:id => @granule_objects.first.collection.id), method: "post") do %>
+              <!-- Todo: Verify AuthenticityToken issue is fixed in RAILS 6.0 -->
```

Comment:
should this not be an ERB comment not an HTML comment:

<%# This is a comment! %>

Reply:
Well there are 71 comments in the ERB files using <!-- so that the seems the convention for this project at least, not sure what is the convention though in general.

Reply:
do the others mention security flaws?